nodeup/pkg/model/kubelet.go

```diff
@@ -145,6 +147,17 @@ func (b *KubeletBuilder) buildSystemdEnvironmentFile(kubeletConfig *kops.Kubelet
 		flags += " --network-plugin-dir=" + b.CNIBinDir()
 	}
 
+	// If a cluster has a container registry set, flag kubelet to use the pause container from that registry.
+	if b.Cluster.Spec.Assets != nil && b.Cluster.Spec.Assets.ContainerRegistry != nil {
+		// TODO another hardcoded version
```

Comment:
My knee-jerk is that it's just another layer of indirection that'll be confusing.  But I'm happy to look at a PR.  It sounds like a separate PR though...